app/models/publication_status.rb

```diff
@@ -0,0 +1,3 @@
+class PublicationStatus
```

Comment:
interested why a model instead of an enum on the publishable concern (e.g. [this][0])

[0]:https://api.rubyonrails.org/v5.2.1/classes/ActiveRecord/Enum.html

Reply:
Because I didn't think of that last night. 🤦🏻‍♀️

I'll change that.

Reply:
Removed this class and moved to an enum on Publishable. Thanks for the suggestion!